src/site/content/en/blog/aurora-resource-inlining/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Resource Inlining in JavaScript frameworks'
```

Comment:
```suggestion
title: 'Resource inlining in JavaScript frameworks'
```

Reply:
@rachelandrew Sentence case for titles and headings https://web.dev/handbook/grammar/#titles-and-headings